PackageInfo.g

```diff
@@ -15,7 +15,7 @@ Version := Maximum( [
                    ## this line prevents merge conflicts
                    "2018.10-18", ## Florian's version
                    ## this line prevents merge conflicts
-                   "2020.05-10", ## Kamal's version
+                   "2021.02-15", ## Kamal's version
```

Comment:
Just for the future. This numbering scheme is a bit different. The last number should reflect the number of the commit in the current month, not a date.